rpcd/playbooks/roles/kibana/tasks/kibana_post_install.yml

```diff
@@ -40,8 +74,8 @@
   template:
     src: "{{ item }}"
     dest: "/etc/apache2/sites-available/{{ item }}"
-    owner: "{{ kibana_system_user }}"
-    group: "{{ kibana_system_group }}"
+    owner: "root"
+    group: "root"
```

Comment:
I'd be enclined to use apache user. I don't really have a problem of root:root, because I guess 644 would be the default chmod and would be fine for apache to read the file.